src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -1750,6 +1753,18 @@ namespace HeatBalanceSurfaceManager {
                                     "Zone",
                                     "State",
                                     Surface(loop).Name);
+                SetupOutputVariable("Surface Outside Face Thermal Radiation to Air Heat Transfer Rate",
+                                    OutputProcessor::Unit::W,
+                                    QAirExtReport(loop),
+                                    "Zone",
```

Comment:
This is correct. The three new surface outputs are the only ones that accumulate at the zone timestep.